packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -35,8 +35,57 @@ class PlaceOrderError extends Error {
     }
 }
 
+class DefaultGetCheckoutError extends Error {
+    constructor (message, errorCode) {
+        super(message);
+        this.messageKey = 'errorMessages.pageLoad.description';
+        this.eventToEmit = EventNames.CheckoutGetFailure;
+        this.logMessage = 'Get Checkout Failure';
+        this.errorCode = errorCode;
+        this.shouldShowInDialog = false;
+        this.errorFormType = 'pageLoad';
```

Comment:
Should we extract this `pageLoad` to the constants file? Since we're using it in many places.